docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -226,6 +226,27 @@ For more information regarding setup of the PostgresSQL connection, see `Postgre
 
      hba
 
+Setting up a MsSQL Database
```

Comment:
Can you also mention in section above - "Choosing database backend"?

Reply:
Yep. Good point.